_episodes_rmd/01-starting-with-data.Rmd

```diff
@@ -430,7 +430,7 @@ We'll learn why this is so in the next lesson.
 > > ## Solution
 > > ~~~
 > > whichPatients <- seq(2,40,2)
-> > whichDays <- c(1:5)
+> > whichDays <- 1:5
```

Comment:
Hmmm. I don't agree (I think this just muddies people's mental models), but OK.  If you wanted to keep it even simpler, you could use `seq(1,5)` here - but then students wouldn't see the colon operator, which is useful (although dangerous) and ubiquitous.